src/api/app/assets/javascripts/webui/request_show_redesign/add_review.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+  $('#add-review-dropdown-component .dropdown-item').on('click', function() {
+    var review = $(this);
+
+    $('#review_form_collapse h5 i').html(review.html());
+    $('#review_id').val(review.data('review'));
+  });
+
+  $('#add-review-dropdown-component').on('shown.bs.dropdown', function () {
+    $('#review_form_collapse').collapse('hide');
+  });
+});
+
+$(document).click(function(e) {
```

Comment:
This will be loaded on all pages, even if they don't have the `review_form_collapse` element. This is not efficient. We should move this JS code to the view which contains this element.

Reply:
done